config/rbac/role_binding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: contour-plus-rolebinding
+  name: manager-rolebinding
```

Comment:
This name seems too generic.
Will this be changed or prefixed when deployed in our environment?

Reply:
I confirmed `namePrefix: contour-plus-` in config/default/kustomization.yaml.